grc/core/generator/flow_graph.tmpl

```diff
@@ -78,18 +78,17 @@ $imp
 #set $class_name = $flow_graph.get_option('id')
 #set $param_str = ', '.join(['self'] + ['%s=%s'%(param.get_id(), param.get_make()) for param in $parameters])
 #if $generate_options == 'wx_gui'
-    #import gtk
-    #set $icon = gtk.IconTheme().lookup_icon('gnuradio-grc', 32, 0)
+import gtk
```

Comment:
This introduces a runtime-dep on gtk into every wxgui flowgraph

Reply:
Oof sorry was copying this change in from hacks in a docker container

Reply:
I honestly don't know whether there's no inherent gtk dependency in wx or its python bindings. If there isn't, we can't do this, I'm afraid.

Reply:
@anuraaga  not quite sure what you mean with "compile time dependency on gtk for all graphs": that doesn't reflect my understanding of what GNU Radio is. Can you elaborate, please?

Reply:
I mean that when compiling a graph, for example using `grcc`, the system must have `gtk` installed, regardless of what the content of the graph is. This is even when compiling a `nogui` graph. After compilation, the resulting python will not depend on `gtk` but it's bad that we can't even compile the graphs without the dependency installed IMO.